src/DataRootTuple.sol

```diff
@@ -7,7 +7,8 @@ pragma solidity ^0.8.4;
 ///  https://github.com/celestiaorg/celestia-specs/blob/master/src/specs/data_structures.md#header
 struct DataRootTuple {
     // Celestia block height the data root was included in.
-    // Genesis block is height = 1.
+    // Genesis block is height = 0.
```

Comment:
One minor nit: It is the genesis file that has height 0. The block at height 1 will reference the genesis file as its parent. It is only semantics but I would argue that the first _block_ is the one produced by consensus and the genesis file does not really count as a block as it is not produced by the same consensus protocol. 

Question: Does the genesis file / the initial chain state play any role here at all? Isn't the first relevant block only that with height 1? If not, can you clarify how the genesis file plays a role in this smart contract?

Reply:
It doesn't play any role in the smart contract, it's just a comment to specify from where the data commitments will start.